1-exercises/J-string-substring/exercise.js

```diff
@@ -6,7 +6,7 @@
 
 let statement = "I like programming and dogs";
 
-statement = statement.substring();
+statement = statement.substring(0, 18);
```

Comment:
😁 Nice...Magic Numbers though..
Can you think of combining calculating the end position of the substring "and dogs" using another string method?

Reply:
I see magic numbers is a common issue for me. Will go through it all and try without them. Thanks